content/automate/ManageIQ/Cloud/VM/Retirement/Email.class/__methods__/vm_retire_extend.rb

```diff
@@ -15,6 +15,7 @@ def initialize(handle = $evm)
               end
 
               def main
+                @handle.log("warn", "[DEPRECATION] This method will be deprecated. Please use similarly named method from System/Notification/Email class.”)
```

Comment:
@billfitzgerald0120 You have the wrong end quotes here, seems like a copy/paste from Mac

https://apple.stackexchange.com/questions/120486/quotes-problem-in-mavericks-or-textedit